src/metaschema/oscal_ssp_metaschema.xml

```diff
@@ -705,6 +706,10 @@
       <field ref="remarks" in-xml="WITH_WRAPPER"/>
     </model>
     <constraint>
+      <allowed-values target="link/@rel" allow-other="yes">
+        &allowed-values-component_component_link-rel;
+        <enum value="imported-from">Identifies the UUID or URI the <code>component-definition</code> that originally defined the <code>component</code>.</enum>
```

Comment:
```suggestion
        <enum value="imported-from">The hyperlink identifies a URI pointing to the <code>component</code> in a <code>component-definition</code> that originally defined the control narrative.</enum>
```

Reply:
Thinking about this more, I believe this link relation should be on the `by-component` instead, since that is where the control narrative should exist. Perhaps we should merge my suggestion above and then relocate it there?